examples/containerlab/.topology.clab.yml.bak

```diff
@@ -0,0 +1,22 @@
+name: bonding
+mgmt:
+  network: inmanta_mgmt
+  ipv4_subnet: 10.0.0.0/24
+topology:
+  nodes:
+    R1:
+      kind: linux
+      image: nfv-test-api:latest
```

Comment:
Here I would actually use the image published here: https://hub.docker.com/layers/inmantaci/nfv-test-api/latest/images/sha256-411f8a5df63ac8fa155d17599139b084761e085b4364f3fe9254dc46f6dbfdb6?context=explore